schemas/v3.1/schema.json

```diff
@@ -799,6 +799,18 @@
           "$ref": "#/$defs/response-or-reference"
         }
       },
+      "if": {
+        "required": [
+          "default"
+        ]
+      },
+      "then": {
+        "$comment": "The Responses Object MUST contain at least one response code",
+        "minProperties": 2
```

Comment:
As the spec language is not clear around this, we suggest that a `default` response code should be considered as meeting the requirement of "at least one response code", therefore the JSON schema should not require 2 properties.

Reply:
okay, fixed!